src/ansys/pytwin/evaluate/tbrom.py

```diff
@@ -229,15 +215,16 @@ def _read_settings(filepath):
 
         # Create list of name selections indexes
         for name, idsList in namedselection.items():
+            idsListNp = np.array(idsList)
+            ind = np.where(idsListNp == -1)
             finallist = []
```

Comment:
We can remove finallist since it's no longer used

Reply:
yes indeed, thanks